<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Of 4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Game mode:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" colspan="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Head to Head</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Human vs PC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PC vs PC</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Battle field size:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QSpinBox" name="fieldSizeX">
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QSpinBox" name="fieldSizeY">
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="5">
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="player1TypeL">
      <property name="text">
       <string>Head 1 =</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="player2TypeL">
      <property name="text">
       <string>Head 2 =</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLineEdit" name="player2ColorLE">
      <property name="text">
       <string>Ao</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="player1ColorLE">
      <property name="text">
       <string>Aka</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="swapB">
      <property name="text">
       <string>swap</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>309</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
